src/conf_mode/system_frr.py

```diff
@@ -19,18 +19,21 @@
 from vyos import ConfigError
 from vyos.base import Warning
 from vyos.config import Config
+from vyos.frrender import FRRender
+from vyos.frrender import get_frrender_dict
 from vyos.logger import syslog
 from vyos.template import render_to_string
 from vyos.utils.boot import boot_configuration_complete
 from vyos.utils.file import read_file
 from vyos.utils.file import write_file
 from vyos.utils.process import call
+from vyos.utils.process import is_systemd_service_running
 
 from vyos import airbag
 airbag.enable()
 
 # path to daemons config and config status files
-config_file = '/etc/frr/daemons'
+deamon_config_file = '/etc/frr/daemons'
```

Comment:
It's spelled `daemon` not `deamon`
```suggestion
daemon_config_file = '/etc/frr/daemons'
```